src/static/css/_pagination.css

```
@@ -0,0 +1,73 @@
+.pagination {
+    align-items: center;
+    background: var(--fl-bgColor, var(--pagination-bg));
+    box-shadow: inset 0 0.2rem var(--fl-linkColor, var(--pagination-box-shadow-color));
+    color: var(--fl-fgColor, var(--pagination-color));
```

Add a third fallback to `--color-blue-600` here so you don't need to define a variable globally for this:

```suggestion
    color: var(--fl-fgColor, var(--pagination-color, var(--color-blue-600)));
```